framework/src/main/java/org/checkerframework/common/reflection/DefaultReflectionResolver.java

```diff
@@ -520,11 +531,7 @@ private boolean isUnknownMethod(MethodInvocationTree tree) {
             if (result.isEmpty()) {
                 debugReflection("Unable to resolve method: " + className + "@" + methodName);
             }
-        } catch (SecurityException
-                | NoSuchMethodException
-                | IllegalAccessException
-                | IllegalArgumentException
-                | InvocationTargetException e) {
+        } catch (SecurityException e) {
```

Comment:
Which of the methods in this large block can throw the `SecurityException`? Make the scope as small as possible.

Reply:
No specific method was throwing `SecurityException`. So, I have removed it in the latest commit.